docs/rules/periphery/periphery-append-units-to-csv.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
```

Comment:
I thought we were going to remove this script?

Reply:
I misunderstood and just removed the Makefile call.
I'm removing it now.

Reply:
@wgryncewicz - Also see my comment on slack about resolving comments when you fix them.